### ISSUE TITLE
multilinguisme

### ISSUE THREAD

**Issue by Fil** [severo/portfolios-redcartografica#4]
je pense que c'est important pour nous …  en es fr it no

deux options:
- petit bouton en haut qui bascule d'une langue à l'autre
- mettre tous les textes ensemble comme dans les cartels de musées (risque de surcharge)

**Comment by Fil** [comment]
Il me semble que le plus simple techniquement parlant mais aussi en terme de navigation serait d'avoir une fiche du genre fil.en.md et un répertoire fil.en/ ; etc.

**Comment by severo** [comment]
Oui, c'est très important. Hugo a tout ce qu'il faut pour gérer ça. Mais il faut décider de la meilleure façon de structurer les URL, et de naviguer.
